src/object_gui.cpp

```diff
@@ -317,10 +418,10 @@ class BuildObjectWindow : public Window {
 	 * Select the specified object class.
 	 * @param object_class_index Object class index to select.
 	 */
-	void SelectOtherClass(ObjectClassID object_class_index)
+	void SelectOtherClass(const ObjectClass *object_class)
```

Comment:
Please update the function documentation.

Reply:
Fixed, resolving.

Reply:
Should that be "Object class **to** select" instead of "Object class select"?